src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/EnvelopeParser.java

```diff
@@ -19,7 +19,7 @@
     public static Envelope parse(byte[] bytes) {
         try {
             return objectMapper.readValue(bytes, Envelope.class);
```

Comment:
Are we handling scenario where bytes are null ? We can put throw custom exception message rather than handling null ?

Reply:
different from `InvalidMessageException`?